include/wx/msw/icon.h

```diff
@@ -69,8 +69,12 @@ class WXDLLIMPEXP_CORE wxIcon : public wxGDIImage
     // implementation only from now on
     wxIconRefData *GetIconData() const { return (wxIconRefData *)m_refData; }
 
-    void SetHICON(WXHICON icon) { SetHandle((WXHANDLE)icon); }
+#if WXWIN_COMPATIBILITY_3_0
+    wxDEPRECATED_INLINE(void SetHICON(WXHICON icon), SetHandle((WXHANDLE)icon); )
+#endif // WXWIN_COMPATIBILITY_3_0
+
     WXHICON GetHICON() const { return (WXHICON)GetHandle(); }
+    void SetHICON(WXHICON icon, int width, int height);
```

Comment:
Of course, if we rename the other method to `InitHBITMAP()`, this one should be renamed ot `InitHICON()` too for consistency.